eth/eventhandler/event_handler_test.go

```diff
@@ -1346,6 +1233,118 @@ func TestHandleBlockEventsStream(t *testing.T) {
 			require.False(t, share.Liquidated)
 		})
 	})
+
+	t.Run("test OperatorRemoved event handle", func(t *testing.T) {
+
+		// Should return MalformedEventError and no changes to the state
+		t.Run("test OperatorRemoved incorrect operator ID", func(t *testing.T) {
+			// Call the contract method
+			_, err = boundContract.SimcontractTransactor.RemoveOperator(auth, 100500)
+			require.NoError(t, err)
+			sim.Commit()
+
+			block := <-logs
+			require.NotEmpty(t, block.Logs)
+			require.Equal(t, ethcommon.HexToHash("0x0e0ba6c2b04de36d6d509ec5bd155c43a9fe862f8052096dd54f3902a74cca3e"), block.Logs[0].Topics[0])
+
+			eventsCh := make(chan executionclient.BlockLogs)
+			go func() {
+				defer close(eventsCh)
+				eventsCh <- block
+			}()
+
+			// Check that there is 1 registered operator
+			operators, err := eh.nodeStorage.ListOperators(nil, 0, 0)
+			require.NoError(t, err)
+			require.Equal(t, len(ops), len(operators))
+
+			// Handle the event
+			lastProcessedBlock, err := eh.HandleBlockEventsStream(eventsCh, false)
+			require.Equal(t, blockNum+1, lastProcessedBlock)
+			require.NoError(t, err)
+			blockNum++
+
+			// Check if the operator wasn't removed successfully
+			operators, err = eh.nodeStorage.ListOperators(nil, 0, 0)
+			require.NoError(t, err)
+			require.Equal(t, len(ops), len(operators))
+		})
+
+		// Receive event, unmarshall, parse, check parse event is not nil or with error, operator id is correct
```

Comment:
This kind of description is probably better split in multiple lines and put those right where these things happen, while test description should probably describe more high-level intent for this test

Reply:
I'd even delete this comment. But I just moved this subtest closer to the end of the file and uncommented the related check. I didn't want to touch any irrelevant code. Also, I think your comment applies to other places in this file, so I think I'd prefer this to be out of the scope of this PR even though I'm touching this one

Reply:
True, you are right.